packages/waggle/src/storage/storageBasic.ts

```diff
@@ -0,0 +1,50 @@
+import Libp2p from 'libp2p'
+import OrbitDB from 'orbit-db'
+import PeerId from 'peer-id'
+import { StorageOptions } from '../common/types'
+import { createPaths } from '../common/utils'
+import logger from '../logger'
+import { Storage } from '../storage'
+
+const log = logger('dbSnap')
+
+class StorageTestSnapshotOptions extends StorageOptions {
+  messagesCount: number
+  createSnapshot?: boolean = false
+  useSnapshot?: boolean = false
+}
+
+export class StorageBasic extends Storage {
```

Comment:
StorageBasic sounds like it would be the base class but here it extends Storage. Is it a base just for tests?

Reply:
Yes it is a base for test, we should create maybe a storageTestUtils and put it inside along with storageSnapshot, because we use them solely for test purposes

Reply:
I didn't do that keep MR clear